app/pages/dashboard/contextCreator.test.js

```diff
@@ -29,4 +48,75 @@ describe('getContext', () => {
+//         dataTestId: `description-${order.orderId}`,
+//       },
+//       {
+//         data: order.lastUpdatedBy,
+//         dataTestId: `lastUpdatedBy-${order.orderId}`,
+//       },
+//       {
+//         data: formatDate(order.lastUpdated),
+//         dataTestId: `lastUpdated-${order.orderId}`,
+//       },
+//       {
+//         data: formatDate(order.dateCreated),
+//         dataTestId: `dateCreated-${order.orderId}`,
+//       },
+//     ];
+//     if (order.status === 'Submitted') acc.submittedOrders.push(formattedOrder);
+//     else acc.unsubmittedOrders.push(formattedOrder);
+//     return acc;
+//   }, { submittedOrders: [], unsubmittedOrders: [] });
+
+//   return ({
+//     submittedOrders: formattedOrders.submittedOrders,
+//     unsubmittedOrders: formattedOrders.unsubmittedOrders,
+//   });
+// };
+  fdescribe('ordersData', () => {
+    it('should format a submitted order correctly', () => {
+      const context = getContext({ orgId: 'Org1', ordersData: mockOrdersData });
+      expect(context.submittedOrders.length).toEqual(1);
+      const submittedOrder1 = context.submittedOrders[0];
+      expect(submittedOrder1[0].data).toEqual('order2');
+      expect(submittedOrder1[0].href).toEqual(`${baseUrl}/organisation/order2`);
+      expect(submittedOrder1[0].dataTestId).toEqual('id-order2');
+      expect(submittedOrder1[1].data).toEqual('Some new order');
+      expect(submittedOrder1[1].dataTestId).toEqual('description-order2');
+      expect(submittedOrder1[2].data).toEqual('Alice Smith');
+      expect(submittedOrder1[2].dataTestId).toEqual('lastUpdatedBy-order2');
+      expect(submittedOrder1[3].data).toEqual('6 June 2020');
+      expect(submittedOrder1[3].dataTestId).toEqual('lastUpdated-order2');
+      expect(submittedOrder1[4].data).toEqual('6 June 2020');
+      expect(submittedOrder1[4].dataTestId).toEqual('dateCreated-order2');
+    });
+
+    it('should format an unsubmitted order correctly', () => {
+      const context = getContext({ orgId: 'Org1', ordersData: mockOrdersData });
+      expect(context.unsubmittedOrders.length).toEqual(1);
+      const unsubmittedOrder1 = context.unsubmittedOrders[0];
+      expect(unsubmittedOrder1[0].data).toEqual('order1');
+      expect(unsubmittedOrder1[0].href).toEqual(`${baseUrl}/organisation/order1`);
+      expect(unsubmittedOrder1[0].dataTestId).toEqual('id-order1');
+      expect(unsubmittedOrder1[1].data).toEqual('Some Order');
+      expect(unsubmittedOrder1[1].dataTestId).toEqual('description-order1');
+      expect(unsubmittedOrder1[2].data).toEqual('Bob Smith');
+      expect(unsubmittedOrder1[2].dataTestId).toEqual('lastUpdatedBy-order1');
+      expect(unsubmittedOrder1[3].data).toEqual('6 May 2020');
+      expect(unsubmittedOrder1[3].dataTestId).toEqual('lastUpdated-order1');
+      expect(unsubmittedOrder1[4].data).toEqual('6 May 2020');
+      expect(unsubmittedOrder1[4].dataTestId).toEqual('dateCreated-order1');
+    });
+  });
```

Comment:
I'd add one more test case with multiple submitted and unsubmitted orders but just check the length of these are what you expect